### DISCUSSION TITLE
VITS avg_loss_0 and avg_loss_disc are identical

### DISCUSSION THREAD

**Discussion by MedAymenF** [idiap/coqui-ai-TTS#170] [Q&A] [answered]
I was training a VITS model when I noticed that the `avg_loss_0` and `avg_loss_disc` loss curves are identical. Is this normal? and if it is, is it not redundant to have both?
[Here](https://huggingface.co/Kamtera/persian-tts-female-vits/tensorboard?params=scalars#frame)'s another training run showing the same thing.
![Screenshot 2024-11-21 at 02-07-52 TensorBoard](https://github.com/user-attachments/assets/c7440984-41b3-4b47-ab09-cf7e1f9d0e0c)

**Comment by eginhard** [comment, marked as answer]
Yeah, they're always equal: https://github.com/idiap/coqui-ai-TTS/blob/9035e36b1f4d59576c749c8f43fc2ad3e7881c0b/TTS/tts/layers/losses.py#L768-L775

Having it named `loss_disc` helps to tell what the loss means. I'm not sure if the other one could be safely removed or if that's a standard name that Coqui expects to be there.

**Reply by MedAymenF** [reply]
Thanks.
